src/js/view/controls/tizen/tizen-controlbar.ts

```diff
@@ -62,6 +66,7 @@ export default class TizenControlbar extends Controlbar {
     activeButton: Button | null;
     el: HTMLElement;
     nextUpToolTip: NextUpToolTip;
+    adSkipButton: any;
```

Comment:
is there any way to explicitly type this?

Reply:
i'm not sure if there is since this component exists in `jwplayer-commercial`